src/Payment.ts

```diff
@@ -212,4 +212,81 @@ export class Payment {
     const params = [ 'paymentrequest', networkAddress, this._user.address, amount, subject ]
     return this._utils.createLink(params)
   }
+
+  /**
+   * Retrieve total spendable amount of user in currency network
+   *
+   * @param networkAddress
+   *
+   * @return {Promise<{amount: Amount}>}
+   */
+  public async getMaxAmount (
+    networkAddress: string
+  ): Promise<any> {
+    const decimals = await this._currencyNetwork.getDecimals(networkAddress)
+    const userAddress = this._user.address
+    const endpoint = `networks/${networkAddress}/users/${userAddress}/spendable`
```

Comment:
This endpoint just gives the available and is deprecated because of the confusing name, as it does not necessarily is the spendable amount because of the fees.

Reply:
I actually thought I removed it already.

Reply:
I opened an issue. The docs are up to date with the source code.. But not with the project.